packages/apollo-language-server/src/fileSet.ts

```diff
@@ -25,6 +27,26 @@ export class FileSet {
     this.rootURI = rootURI;
     this.includes = includes;
     this.excludes = excludes;
+
+    /**
+     * This function is used in the Array.filter function below it to remove any .env files and config files.
+     * If there are 0 files remaining after removing those files, we should warn the user that their config
+     * may be wrong. We shouldn't throw an error here, since they could just be initially setting up a project
+     * and there's no way to know for sure that there _should_ be files.
+     */
+    const filterConfigAndEnvFiles = (path: string) =>
+      !(
+        path.match(/.*(\.env|apollo\.config\.js).*/) ||
```

Comment:
Just for simplicity, I'll suggest:
`path.includes("apollo.config") || path.includes(".env")`

Just a minor preference, and not a big deal if you prefer the regex.

We might not account for this in all places, but we should allow `apollo.config.ts`.